console/src/main/java/org/georchestra/console/ws/changepassword/ChangePasswordFormController.java

```diff
@@ -93,8 +94,8 @@ public String setupForm(HttpServletRequest request, HttpServletResponse response
             throws DataServiceException {
         Optional<String> uid = getUsername();
         if (uid.isPresent()) {
-            boolean isExternalAuth = Boolean
-                    .parseBoolean(SecurityHeaders.decode(request.getHeader(SEC_EXTERNAL_AUTHENTICATION)));
+            boolean isExternalAuth = !Objects.isNull(request.getHeader(SEC_EXTERNAL_AUTHENTICATION))
```

Comment:
you can use `Objects.nonNull` : https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#nonNull-java.lang.Object-